app/screens/sso/sso.js

```diff
@@ -168,12 +192,12 @@ class SSO extends PureComponent {
                     source={{uri: this.loginUrl}}
                     javaScriptEnabledAndroid={true}
                     automaticallyAdjustContentInsets={false}
-                    scalesPageToFit={true}
+                    scalesPageToFit={false}
```

Comment:
Does this causes changes in the way other sso pages display?

Reply:
Moved this setting to state to make sure it only changes if SSO is OneLogin